docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,171 @@
+1. Triggering outbox messages: After the commitment is cemented, any client can trigger operations from the Smart Rollup.
+
+Here is more information on each of these phases:
+
+{/* TODO diagram of commitment periods and refutation periods? */}
+
+### Origination
+
+Like smart contracts, users deploy Smart Rollups to layer 1 in a process called _origination_.
+
+The origination process stores data about the rollup on layer 1, including:
+
+- An address for the rollup, which starts with `sr1`
+- The type of proof-generating virtual machine (PVM) for the rollup, which defines the execution engine of the rollup kernel; currently only the `wasm_2_0_0` PVM is supported
+- The code of the rollup, which is a WebAssembly program referred to as the rollup's _kernel_
+- The Michelson data type of the messages it receives from layer 1
+- The genesis commitment that forms the basis for commitments that rollups nodes publish in the future
+
+After it is originated, anyone can run a Smart Rollup node based on this information.
+
+### Commitment periods
+
+Starting from the rollup origination level, levels are partitioned into _commitment periods_ of 60 consecutive layer 1 blocks.
+During each commitment period, the each rollup node receives the messages in the rollup inbox, processes them, and updates its state.
+
+Because Smart Rollup nodes behave in a deterministic manner, their states should all be the same if they have processed the same inbox messages with the same kernel starting from the same origination level.
+This state is referred to as the "state of the rollup."
+
+At the end of the commitment period, each rollup node must publish a hash of its state to layer 1, which is called its _commitment_.
+Each commitment builds on the previous commitment, and so on, back to the genesis commitment from when the Smart Rollup was originated.
+
+Nodes must stake 10,000 tez along with their commitments.
+When nodes make identical commitments, their stakes are combined into a single stake for the commitment.
+
+Then a refutation period for the commitment period begins, and a new commitment period begins.
+
+### Refutation periods
+
+Because the PVM is deterministic and all of the inputs are the same for all nodes, any honest node that runs the same Smart Rollup produces the same commitment.
+As long as nodes publish matching commitments, they continue running normally.
+
+During the refutation period for a commitment period, if two or more nodes publish different commitments, two of them play a _refutation game_ to identify the correct commitment.
+The nodes automatically play the refutation game by stepping through their logic to identify the point at which they differ.
+At this point, layer 1 uses the PVM to identify the correct commitment.
+Then the protocol gives half of the incorrect commitment's stake to the correct commitment's stake, and burns the other half.
```

Comment:
What's the "any" in "if any?" Does this mean that perhaps both commitments are incorrect so if neither of them win the refutation game, 100% of both of their stakes are burned because there is no correct commitment to give it to?

Reply:
Yes. In the reference doc:

> There are only two possible outcomes: either one of the staker has provided a valid proof, then that staker wins the game, and is rewarded with half of the opponent’s deposit (the other half being burnt); or, both stakers have provided an invalid proof and they both lose their deposit.

Reply:
Updated with this list, which is a little wordy but clear:

Each refutation game has one of two results:

- Neither commitment is correct.
In this case, the protocol burns both commitments' stakes and eliminates both commitments.

- One commitment is correct and the other is not.
In this case, the protocol eliminates the incorrect commitment, burns half of the incorrect commitment's stake, and gives the other half to the correct commitment's stake.